lab-judy/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "lab-11-12-express-api",
+  "version": "1.0.0",
+  "description": "![cf](https://i.imgur.com/7v5ASc8.png) lab 11 single resource express api ======",
+  "main": "index.js",
+  "scripts": {
+    "start": "DEBUG='note*' node server.js",
```

Comment:
if you change your debug require statements to use the name of your model (which is a best practice), remember to change your scripts to reflect that.